packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -40,6 +86,215 @@ const allMemberTypes = ['field', 'method', 'constructor'].reduce<string[]>(
   [],
 );
 
+const neverConfig = {
+  type: 'string',
+  enum: ['never'],
+};
+
+const allMemberTypesArrayConfig = {
+  type: 'array',
+  items: {
+    enum: allMemberTypes,
+  },
+};
+
+const allMemberTypesDefaultConfig = {
+  type: 'string',
+  enum: ['never'],
+};
+
+const allMemberTypesObjectConfig = {
+  type: 'object',
+  properties: {
+    memberTypes: {
+      oneOf: [allMemberTypesDefaultConfig, allMemberTypesArrayConfig],
+    },
+    order: {
+      type: 'string',
+      enum: ['alphabetically'],
```

Comment:
I would consider adding an extra string like `'as-written'`.
Right now it kind-of acts like a boolean, and turning the option on/off by excluding/including it feels cumbersome.

Reply:
I will change this once we've decided on the final API

Reply:
Added 'as-written'